etl/steps/data/garden/animal_welfare/2023-08-14/number_of_farmed_fish.meta.yml

```diff
@@ -0,0 +1,36 @@
+definitions:
+  common: |
+    This study estimates the annual number of farmed fish slaughtered for food using data on aquaculture production tonnages (from the Food and Agriculture Organization of the United Nations) and mean weights estimated from available fish size data (from a range of internet sources).
+
+    Note that this estimate does not include:
```

Comment:
Should "this study" be better?

Reply:
Given your next point, I've realised that it's better to keep using "estimate". The study (or the dataset) does actually include non-fish species. However, estimates on the number of fish is not included for non-fish species.